$NetBSD: patch-.env.template,v 1.2 2024/12/01 00:04:58 pin Exp $

Set for package defaults: Use sqlite, and the log directory we set up

--- .env.template.orig	2024-11-30 22:34:24.970004290 +0000
+++ .env.template
@@ -31,8 +31,8 @@
 # RELOAD_TEMPLATES=false
 
 ## Web vault settings
-# WEB_VAULT_FOLDER=web-vault/
-# WEB_VAULT_ENABLED=true
+WEB_VAULT_FOLDER=@PREFIX@/@WEB_VAULT_DIR@/
+WEB_VAULT_ENABLED=true
 
 #########################
 ### Database settings ###
@@ -40,7 +40,7 @@
 
 ## Database URL
 ## When using SQLite, this is the path to the DB file, default to %DATA_FOLDER%/db.sqlite3
-# DATABASE_URL=data/db.sqlite3
+DATABASE_URL=data/db.sqlite3
 ## When using MySQL, specify an appropriate connection URI.
 ## Details: https://docs.diesel.rs/2.1.x/diesel/mysql/struct.MysqlConnection.html
 # DATABASE_URL=mysql://user:password@host[:port]/database_name
@@ -370,7 +370,7 @@
 # USE_SYSLOG=false
 
 ## Logging to file
-# LOG_FILE=/path/to/log
+LOG_FILE=@VAULTWARDEN_LOGDIR@/vaultwarden.log
 
 ## Log level
 ## Change the verbosity of the log output
